cluster/gce/trusty/master.yaml

```diff
@@ -213,6 +213,7 @@ script
 	set -o errexit
 	set -o nounset
 
+	. /etc/kube-configure-helper.sh
```

Comment:
Yes, kube-addons were moved into a static pod kube-addon manager since release-1.3, but in release-1.2 they are still managed by the old fashion scripts. This should be the reason why you see they are quite different.